sda-commons-web-autoconfigure/src/main/java/org/sdase/commons/spring/boot/web/metadata/MetadataClientInterceptor.java

```diff
@@ -7,40 +7,36 @@
  */
 package org.sdase.commons.spring.boot.web.metadata;
 
-import java.io.IOException;
+import feign.RequestInterceptor;
+import feign.RequestTemplate;
 import java.util.List;
 import java.util.Set;
-import javax.ws.rs.client.ClientRequestContext;
-import javax.ws.rs.client.ClientRequestFilter;
 import org.apache.commons.lang3.StringUtils;
 
-/**
- * A {@link ClientRequestFilter} to submit the {@link MetadataContext} to other services that are
- * called synchronously.
- */
-public class MetadataContextClientRequestFilter implements ClientRequestFilter {
+public class MetadataClientInterceptor implements RequestInterceptor {
```

Comment:
Can we use `MetadataContext` as prefix in all cases?
I.e. from `MetadataClientInterceptor` to `MetadataContextClientInterceptor`

Reply:
We should also keep the documentation.